ark/utils/deepcell_service_utils.py

```diff
@@ -0,0 +1,93 @@
+from twisted.internet import reactor
+from kiosk_client import manager
+import os
+import glob
+from zipfile import ZipFile
+import warnings
+
+
+def create_deepcell_output(deepcell_input_dir, points, deepcell_output_dir,
+                           suffix='_feature_0', host='https://deepcell.org', job_type='multiplex'):
+    """ Handles all of the necessary data manipulation for running deepcell tasks.
+
+        Creates .zip files (to be used as input for DeepCell),
+        calls run_deepcell_task method,
+        and extracts zipped output files to the specified output location
+
+        Args:
+            points (list):
+                List of points in preprocessing pipeline
```

Comment:
I think an option to set `points=None` so that all `.tif` files in the input directory can be read in would be pretty useful.  We have some tools written to help with this.  If you import `ark.utils.io_utils`, you could use something like:
```
if points is None:
    tifs = io_utils.list_files(deepcell_input_dir, substrs='.tif')
    points = io_utils.extract_delimited_names(tifs, delimiter='.')
```
That should plug directly into the code you've already written, assuming I didn't make any typos, etc.

Reply:
added in the last commit